neco-server-exporter/pkg/exporter/helper.go

```diff
@@ -0,0 +1,18 @@
+package exporter
+
+import (
+	"fmt"
+	"maps"
+	"slices"
+)
+
+func GetMetricsName(name string, labels map[string]string) string {
```

Comment:
suggestion: `GetMetricsName` might be better as `BuildMetricName` since it's constructing a new string rather than retrieving an existing one. The `Get` prefix usually suggests fetching something that already exists, which could be slightly misleading here.